src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+function getNumberFromSalary(item) {
+  const result = +[...item.dataset.salary].map((char) => {
+    if (isNaN(char)) {
+      return;
+    }
+
+    return char;
+  }).join('');
+
+  return result;
+}
```

Comment:
> This can be written in better way. For example, you can try to do sth like that:
> 
> ```
> function getNumberFromSalary(item) {
>     return parseInt(item.dataset.salary.replace(/[$,]/g, ''));
> }
> ```

Regexp not studied yet